.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,12 @@
+<!-- Thanks for taking the time to report your issue with _OHHTTPStubs_! When submitting your issue, please make sure to check the following boxes by putting an x in the appropriate [ ] so we can fully understand the context of your problem and help you better -->
```

Comment:
Now that I'm reading that again, the part "putting an x in the appropriate [ ]" feels like it's singular ("in the appropriate box") while we actually want to mean "in the appropriate boxes" — as the contributor should not limit their choices to only one box. Maybe changing the formulation to "putting an x in each appropriate [ ]" would help make that clearer?